src/rlx_goal.erl

```diff
@@ -16,9 +16,9 @@
 file(Filename) -> case file:read_file(Filename) of {ok,Bin} -> parse(Bin); Err -> Err end.
 
 -spec parse(binary() | list()) -> any().
-parse(List) when is_list(List) -> parse(list_to_binary(List));
+parse(List) when is_list(List) -> parse(unicode:characters_to_binary(List));
 parse(Input) when is_binary(Input) ->
-  setup_memo(),
+  _ = setup_memo(),
```

Comment:
This is generated code. But it just tells the tools to not warn that you aren't using a result of a function by making it explicit you don't care.